### PR TITLE
[Grid] Parse parameters of grid driver configuration

### PR DESCRIPTION
**Pull request by pjedrzejewski**
| Q               | A
| --------------- | ---
| Bug fix?        | no
| New feature?    | yes
| BC breaks?      | no
| Related tickets | -
| License         | MIT

### CODE REVIEW

File: src/Sylius/Bundle/ResourceBundle/Controller/ParametersParserInterface.php [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+use Symfony\Component\HttpFoundation\Request;
+
+/**
+ * @author Paweł Jędrzejewski <pawel@sylius.org>
+ */
+interface ParametersParserInterface
+{
+    /**
+     * @param array   $parameters
```
Comment (lchrusciel) [inline src/Sylius/Bundle/ResourceBundle/Controller/ParametersParserInterface.php]: We don't align parameter vertically ;)

File: src/Sylius/Component/Grid/Definition/Grid.php [hunk 2]
```diff
@@ -99,6 +99,14 @@ public function getDriverConfiguration()
     }
 
     /**
+     * @param array $driverConfiguration
+     */
+    public function setDriverConfiguration($driverConfiguration)
```
Comment (lchrusciel) [inline src/Sylius/Component/Grid/Definition/Grid.php]: missing typehint

File: src/Sylius/Bundle/CoreBundle/Doctrine/ORM/PaymentMethodRepository.php [hunk 3]
```diff
@@ -18,6 +18,19 @@ class PaymentMethodRepository extends BasePaymentMethodRepository
     /**
      * {@inheritdoc}
      */
+    public function createListQueryBuilder()
+    {
+        $queryBuilder = $this->createQueryBuilder('o');
```
Comment (Zales0123) [inline src/Sylius/Bundle/CoreBundle/Doctrine/ORM/PaymentMethodRepository.php]: Must this variable be set, or can it just be:
```php
return $this
    ->createQueryBuilder('o')
    ->addSelect('translation')
    ->leftJoin('o.translations', 'translation')
;
```
?

File: src/Sylius/Bundle/ResourceBundle/spec/Controller/ParametersParserSpec.php [hunk 4]
```diff
@@ -25,6 +26,11 @@ function let(ExpressionLanguage $expression)
     {
         $this->beConstructedWith($expression);
     }
+    
+    function it_implements_an_interface()
```
Comment (Zales0123) [inline src/Sylius/Bundle/ResourceBundle/spec/Controller/ParametersParserSpec.php]: Maybe something more specific like ``it_implements_parameters_parser_interface`` or ``it_is_parameters_parser``?

**Comment by michalmarcinkowski** [comment]
Good job! 👍

**Pull request merged**